pkg/backend/httpstate/backend.go

```diff
@@ -1155,12 +1155,26 @@ func (b *cloudBackend) Update(ctx context.Context, stack backend.Stack,
 	return backend.PreviewThenPromptThenExecute(ctx, apitype.UpdateUpdate, stack, op, b.apply, b)
 }
 
-func (b *cloudBackend) IsCopilotFeatureEnabled(opts display.Options) bool {
+// IsCopilotFeatureEnabled implements the "explainer" interface.
+// Checks that the backend supports the CopilotExplainPreviewV1 capability.
+func (b *cloudBackend) IsCopilotFeatureEnabled(ctx context.Context, opts display.Options) bool {
```

Comment:
`IsCopilotFeatureEnabled` is now named wrongly: it should be `IsCopilotExplainPreviewV1Enabled`. Alternatively, export the one with the `apitype` parameter directly.

Reply:
Agree, its implementing a slightly more generic iface so I'd go for `IsExplainPreviewEnabled` wdyt?